AxonDeepSeg/apply_model.py

```diff
@@ -8,7 +8,7 @@
 # AxonDeepSeg imports
 from AxonDeepSeg.visualization.merge_masks import merge_masks
 from AxonDeepSeg import ads_utils
-from config import nnunet_suffix
+from AxonDeepSeg.params import nnunet_suffix
 from AxonDeepSeg.params import intensity
```

Comment:
In this file, you could group the two `AxonDeepSeg.params` import together

Reply:
In some other files, it is also imported twice, but it's a bit different. They will have `from AxonDeepSeg.params import ...` and another line will have `import AxonDeepSeg.params`.

In those cases, it might be easier to keep it the way they are.

Reply:
Done in [34f8ca8](https://github.com/axondeepseg/axondeepseg/pull/844/commits/34f8ca8551e6b6843b6ceb8afbaf6b46a7ddaec3) !